server/src/app.controller.ts

```diff
@@ -11,4 +11,9 @@ export class AppController {
   async login(@Request() req) {
     return this.authService.login(req.user);
   }
+
+  @Get('healthy')
+  async checkServer() {
```

Comment:
can we return node SDK version and milvus version?

Reply:
It's only check the insight server.  We don't need to connect milvus at this condition. 
So we can only get node SDK version.

Reply:
yeah, just return node sdk version here for now.